packages/cli/src/lib/helpers/wrap.ts

```diff
@@ -31,10 +37,16 @@ const run = async (
 export const generateWrapFile = async (
   abi: WrapAbi,
   name: string,
-  type: "interface" | "wasm" | "plugin",
+  type: string,
   path: string,
   logger: Logger
 ): Promise<void> => {
+  if (supportedWrapTypes.indexOf(type as WrapManifest["type"]) === -1) {
```

Comment:
minor nit: I find more cleaner to use `find`/`includes` (in this case, `includes`) rather than `indexOf`

Reply:
Agree, gonna fix it up.